akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/AsyncTestingExampleSpec.scala

```diff
@@ -27,7 +27,7 @@ import scala.util.Success
 import scala.util.Try
 
 object AsyncTestingExampleSpec {
-  //#under-test
+  // #under-test
```

Comment:
those comments are referenced by the documentation to import code, need to check this doesn't break with the reformatting

Reply:
If you are talking about sbt-paradox, it doesn't break anything. I have done the same changes to other upstream Akka projects and sbt-paradox still worked fine.